### ISSUE TITLE
Pressing 'Play' and 'Open Code file' by Enter key does not work

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-microbit#948]
**Repro steps:**
1. Open https://makecode.microbit.org/beta#editor
2. Drag one block and right click on a block, select 'Help'
3. Navigate by Tab key on Help page
3. Select 'Play' or 'Open Code File' button by Enter key

**Expected result:**
'Play' or 'Open Code File' button should work normally by Enter key

**Actual result:**
Pressing 'Play' and 'Open Code file' by Enter key does not work
![issue71101](https://user-images.githubusercontent.com/23466737/42564703-0afce09c-8534-11e8-86ad-312b751fff4f.gif)

**Comment by AimeeCao18** [comment]
The bug still repro
